README.md

```diff
@@ -14,6 +14,17 @@ The Ada library passes the full range of tests from the specification,
 across a wide range of platforms (e.g., Windows, Linux, macOS). It fully
 supports the relevant [Unicode Technical Standard](https://www.unicode.org/reports/tr46/#ToUnicode).
 
+A common use of a URL parser is to take a URL string and normalize it. 
```

Comment:
We also need to update the website as well

Reply:
Sure. We should.